src/lib.rs

```diff
@@ -93,6 +93,9 @@ impl Cache {
         // Remove `"hub"`
         path.pop();
         path.push("token");
+        if !path.exists() {
+            log::warn!("File with your token is absent by path {:?}", path);
```

Comment:
```suggestion
            log::info!("Token file not found {:?}", path);
```

`warn` -> `info`. It's perfectly normal to be missing the token file, therefore warn is not appropriate imo.
Also shortened a bit the message.